driver-core/src/main/com/mongodb/operation/QueryBatchCursor.java

```diff
@@ -214,6 +216,17 @@ public ServerAddress getServerAddress() {
         return serverAddress;
     }
 
+    @Override
+    public BsonDocument getResumeToken() {
+        LOGGER.warn("--- Calling QueryBatchCursor.getResumeToken == null");
```

Comment:
Do we need a `getResumeToken` method on BatchCursor or can we get away with just adding it to MongoChangeStreamCursor?

Reply:
The method has been removed.